openbb_platform/providers/tiingo/openbb_tiingo/models/trailing_dividend_yield.py

```diff
@@ -60,4 +60,5 @@ def transform_data(
         **kwargs: Any,
     ) -> List[TiingoTrailingDivYieldData]:
         """Return the transformed data."""
+        data = data[-query.limit :] if query.limit else data
```

Comment:
Just double checking that this is deliberate (the `-`) to make sure we return the 252 _most recent_ days? So Tiingo returns data in chronological order (i.e. most-recent rows are last?)

Reply:
Yes, it's deliberate! That's exactly that.